conf/gunicorn.py

```diff
@@ -28,3 +29,17 @@
 errorlog = "-"
 accesslog = "-"
 preload_app = False
+
+
+def post_fork(server, worker):
+    """
+    Put the worker_id into an env variable for further use within the app.
+    """
+    os.environ["APP_WORKER_ID"] = str(worker.age)
```

Comment:
If I had a choice, I would prefer `GUNICORN_WORKER_ID` for that.

Why do you use `worker.age` instead of `worker.id`?

`worker.age` is a number that indicates how many requests the worker process has handled since it was started.